pex/testing.py

```diff
@@ -137,8 +137,8 @@ def make_project(
             )
             """
         ),
-        "scripts/hello_world": '#!/usr/bin/env python\nprint("hello world!")\n',
-        "scripts/shell_script": "#!/usr/bin/env bash\necho hello world\n",
+        "scripts/hello_world": '#!/usr/bin/env python\nprint("hello world from py script!")\n',
+        "scripts/shell_script": "#!/usr/bin/env bash\necho hello world from shell script\n",
```

Comment:
I changed the output wording slightly, so we can be sure the output comes from the correct source.
Would've liked to change the print in `do_something()` as well, but as I had issues running the tests, I didn't want to mess too much up.

Reply:
What issues did you hit? Issues running tests can point to bugs in Pex itself and those are useful to know about.

Reply:
#1380 for one. But there were more. Mostly due to my environment, I'd suspect. We can take a deep dive in this when I get back from vacation, if you'd like :)